### ISSUE TITLE
Slack ToS changes

### ISSUE THREAD

**Issue by captn3m0** [sdslabs/slack-lens#3]
See Section `4.6` on the new [Slack API ToS](https://slack.com/terms-of-service/api):

> You may not use the Slack API to replicate or compete with core products or services offered by Slack. You acknowledge and agree that Slack has or may in the future offer products or services that are similar to your Application, and nothing will prevent Slack from doing so;

We are replicating a core slack feature (history search).

**Comment by apsdehal** [comment]
We are not going to publicize this then.

**Comment by captn3m0** [comment]
btw, there is a way to log private channels, which is by using the RTM bot api, but then the bot has to be invited in each channel for it to be logged.

**Comment by apsdehal** [comment]
Yup, but it won't make sense. What is the meaning of private group then :p
